docs/coding-guidelines/project-guidelines.md

```diff
@@ -97,6 +97,44 @@ When building an individual project the `BuildTargetFramework` and `TargetOS` wi
 - .NET Framework latest -> `$(NetFrameworkCurrent)-Windows_NT`
 
 # Library project guidelines
+
+## TargetFramework conditions
```

Comment:
Should we call out that `$(TargetFramework)` should be avoided in PropertyGroup conditions (unless those are in a second PropertyGroup)?

Reply:
Thanks, done :)

Reply:
Do you know if we're going to validate that?  I recall at one point we talked about it.

Reply:
I would prefer to not add infra for that and instead push the project system / msbuild team to make progress on the actual issue.